v1/connector/connector.go

```diff
@@ -30,13 +30,16 @@ type Connector struct {
 	ConfigID        string          `json:"configId,omitempty" yaml:"configId,omitempty"`
 	ConfigAuth      string          `json:"configAuth,omitempty" yaml:"configAuth,omitempty"`
 	EnablePCIB      bool            `json:"enablePCIB,omitempty" yaml:"enablePCIB,omitempty"`
+	MID             string          `json:"mid,omitempty" yaml:"mid,omitempty"`
```

Comment:
this shouldnt be defined here.  It will be different per provider. e.g. one connector may use their account ID, another may use a different identifier.